packages/SwingSet/src/vats/comms/state.js

```diff
@@ -238,14 +238,15 @@ export function makeStateKit(state) {
     assert(!p.resolved, X`${vpid} was already resolved`);
   }
 
-  function markPromiseAsResolved(vpid, resolution) {
+  function markPromiseAsResolved(vpid, rejected, data) {
     const p = state.promiseTable.get(vpid);
     assert(p, X`unknown ${vpid}`);
     assert(!p.resolved);
-    insistCapData(resolution.data);
+    insistCapData(data);
```

Comment:
let's add an assert that `rejected` is a boolean, this seems like a pretty good bang-for-the-buck place for one

Reply:
Done.